src/components/SentEmailList/index.jsx

```diff
@@ -0,0 +1,74 @@
+import SentEmailItem from "../SentEmailItem";
+import { useEffect, useState } from "react";
+import SentEmailView from "../SentEmailView";
+
+function SentEmailList() {
+  const [sentEmailArray, setSentEmailArray] = useState(false);
+  const [refreshJson, setRefreshJson] = useState(false);
+  const [sentCurrentID, setSentCurrentID] = useState(0);
+  const [emailItemWidth, setEmailItemWidth] = useState("w-full");
+  const [emailViewWidth, setEmailViewWidth] = useState("w-0");
+  const [buttonClass, setButtonClass] = useState("hidden");
+
+  useEffect(() => {
+    fetch("https://email-client-api.dev.io-academy.uk/emails/sent")
```

Comment:
I have a feeling this component is exactly the same as `EmailList` except for this line. 

Can you think of a better way to handle this?